src/resolution_functions/instrument.py

```diff
@@ -630,40 +674,124 @@ def available_models(self) -> list[str]:
         return list(self._models.keys())
 
     @property
-    def available_models_and_configurations(self) -> dict[str, list[str]]:
+    def available_unique_models(self) -> list[str]:
         """
-        A dictionary mapping each available :term:`model` to the user
-        :term:`configurations<configuration>` available for that :term:`model`.
+        A list of all unique :term:`models<model>` available for this :term:`version` of this
+        :term:`instrument`.
 
-        All :term:`models<model>` available for this :term:`version` of this :term:`instrument`,
-        and all their :term:`configurations<configuration>` are listed.
+        Only includes the versioned models, i.e. lists all versions of all models, but does not list
+        the recommended models.
 
         Returns
         -------
-        models_and_configurations
-            All models and all their configurations.
+        available_models
+            A list of all available unique models.
         """
-        return {model_name: list(model['configurations'].keys())
-                for model_name, model in self._models.items()}
+        return [name for name, value in self._models.items() if not isinstance(value, str)]
 
-    @property
-    def all_available_models_options(self) -> dict[str, dict[str, list[str]]]:
+    def format_available_models_and_configurations(self) -> str:
         """
-        A dictionary mapping each available :term:`model`, to the user
-        :term:`configurations<configuration>` and their :term:`options<option>`.
+        Formats all available :term:`models<model>` and :term:`configurations<configuration>` into a
+        table.
 
-        All :term:`models<model>` available for this :term:`version` of this :term:`instrument`, all
-        the :term:`configurations<configuration>` of each of the :term:`models<model>`, and all the
-        :term:`options<option>` for each of the :term:`configurations<configuration>`, are listed.
+        The table shows each :term:`model` and either which other :term:`model` it is an alias for,
+        or all the :term:`configurations<configuration>` required by that :term:`model`.
 
         Returns
         -------
-        everything
-            All models, all their configurations, and all their options.
+        str
+            A string containing the nicely formatted table.
+        """
+        contents = [['MODEL', 'ALIAS FOR', 'CONFIGURATIONS']]
+        longest_name, longest_alias, longest_config = 5, 9, 14
```

Comment:
Makes it more obvious what these are.
```suggestion
        longest_name, longest_alias, longest_config = ​map(len, contents[0])
```